Changelog.md

```diff
@@ -7,6 +7,7 @@ Important Bugfixes:
 Language Features:
  * Code Generation: Implement copying recursive structs from storage to memory.
  * ABIEncoderV2: Implement encoding of calldata arrays and structs.
+ * Yul: Disallow function definitions inside for-loop init blocks.
```

Comment:
Would it make sense to also disallow function definition inside for-loop post blocks?

Reply:
post blocks do not have irregular scoping rules, they are just as any other block.